app/javascript/packages/document-capture/components/documents-step.jsx

```diff
@@ -35,11 +36,34 @@ function DocumentsStep({
   errors = [],
   registerField = () => undefined,
 }) {
-  const { t } = useI18n();
+  const { t, formatHTML } = useI18n();
   const { isMobile } = useContext(DeviceContext);
+  const serviceProvider = useContext(ServiceProviderContext);
 
   return (
     <>
+      <p className="sm-hide">{t('doc_auth.info.upload_mobile')}</p>
```

Comment:
I don't have particularly strong feelings about using `DeviceContext`'s `isMobile` property here, but worth noting:

1. It isn't quite the same as just a media query, since it considers camera capability ([source](https://github.com/18F/identity-idp/blob/295662befd7e6027a4397eec7340fe03638a809c/app/javascript/packs/document-capture.jsx#L94-L97)). This becomes relevant for things like whether we optimize actions for assuming that "Take Photo" would launch the Acuant SDK experience, vs. a standard file upload input.
2. It's also primarily how we control rendering differences from the tests, by assigning device context one way or the other depending on the type of view we want to emulate.

I think if we're to use a class, I'd at least eventually like to see `sm-hide` go away altogether, in favor of [responsive display classes](https://designsystem.digital.gov/utilities/display/#advanced-settings) (`display-none desktop:display-block`). We're sometimes blocked by LG-3863 for this, due to how BassCSS uses `!important` for its `display-none` class.

Reply:
This was the spot I was most unsure about. I think replacing the `sm-hide` with `isMobile` makes the most sense.